src/interfaces/store-interfaces/state-interfaces/dissolution-state-interface.ts

```diff
@@ -5,6 +5,7 @@ export interface DissolutionStateIF {
   dissolutionType: DissolutionTypes
   dissolutionDate: string
   dissolutionStatementStep: DissolutionStatementIF
+  dissolutionDateFieldError: string
```

Comment:
It would be nicer if the property name matched the getter and action/mutation.

Re:
getDissolutionDateError
setDissolutionDateError
mutateDissolutionDateError

Reply:
Obsolete comment.